aten/src/ATen/native/quantized/cpu/qnnpack/src/q8dwconv/mp8x25-neon-per-channel.c

```diff
@@ -0,0 +1,939 @@
+/*
```

Comment:
I assume these files are mostly the same as their per tensor counter-parts but with some changes?  Could you mark those places to make the review easier?  Also I'm tempted to say I wish the coding style was left uncompliant like the original since that was easier to follow. :)

Reply:
> Also I'm tempted to say I wish the coding style was left uncompliant like the original since that was easier to follow. :)

Not sure if I follow what you mean.

I will simply paste the diff between the files in the comments section.

Reply:
By the way as a note aside. Current method of depthwise conv for per channel requires us to maintain two pointers zero point and requant scale. This requires us to do some stack manipulation to save these pointers. I did experiment with another approach where zero point and requant scales are packed with weights. It did not much difference for gemm/conv kernels, but for per channel depthwise I think it makes about 12% difference for aarch32 assembly kernel. But doing this only for depthwise requires some changes that I think I will do in a later PR.

Reply:
Very nice.